### DISCUSSION TITLE
Tooltip needs border-radius: 4px;

### DISCUSSION THREAD

**Discussion by dustinmazon** [RedHat-UX/red-hat-design-system#1839] [Design library feedback]
@coreyvickery @marionnegp I am currently doing QA on a personalization project and noticed that the developed version of the tooltip element is missing border radius on all 4 corners of the tooltip container. I am basing this request on the Figma component for tooltip and the previews of the tooltip on [ux.redhat.com ](https://ux.redhat.com/elements/tooltip/#overview) From what I can tell it looks like the visual design examples in documentation and Figma don't match up with what is in the code base or in the demo on UX dot. Could we please get that added during the next release cycle? I mean why be square when you could round up to the coolest style? ;)

Visual example: https://ux.redhat.com/elements/tooltip/#overview
Demo example: https://ux.redhat.com/elements/tooltip/demos/

**Comment by marionnegp** [comment]
Lol, @dustinmazon. I did an audit of our border radius properties in Figma last week to see where we're inconsistent with our current token values or the repo. I do like having rounded corners for tooltips and popovers (once we get that developed), so I'll go ahead and open an issue for this feedback. Thanks!

**Reply by marionnegp** [reply]
I opened [an issue](https://github.com/RedHat-UX/red-hat-design-system/issues/1840) if you're interested in following along.

**Reply by coreyvickery** [reply]
@dustinmazon Steven is handling code changes and the Figma library has been updated.

**Comment by marionnegp** [comment]
This was fixed by #1840
